.mergify.yml

```diff
@@ -13,3 +13,17 @@ pull_request_rules:
     actions:
       queue:
         name: default
+        
+  - name: Automatic merge on approval (Picasso Runtime)
+    conditions:
+      - or:
+        - "#approved-reviews-by>=3"     
+        - and:
+          - "#approved-reviews-by>=2"
+          - approved-reviews-by=@ComposableFi/core
+      - base=main    
+      - files ~= ^code\/parachain\/runtime\/picasso\/(lib\.rs|governance\.rs)
+      - -title~="#wip"
```

Comment:
What does this line do?

Reply:
it avoids merge for wip hash tagged PRs:

https://en.wikipedia.org/wiki/Hashtag

Reply:
Actually it does same it did for other flow (10 lines above this #wip) , @vimukthi-git please look into whole small mergify yaml file. Imho it is hard to review if you do not look into what exists (there was already wip filter). Please skim over for DSL https://docs.mergify.com/conditions/ if something unclear.